### ISSUE TITLE
need more details on how to use it

### ISSUE THREAD

**Issue by bonbonsiregar** [mark3labs/mcp-filesystem-server#1]
Especially on how to use the commands like read_file, write_file, list_allowed_directories, and etc. Also on how to integrate it into Claude Desktop.

**Comment by ratacat** [comment]
Yeah I'm having trouble getting the server to actually run. Installed it in roo code, but when I try and run it nothing happens...im guessing it's just hanging for some reason

 eterna % ps aux | grep mcp-filesystem-server
       65231   0.0  0.0 408102624   1248 s045  S+    2:36PM   0:00.00 grep mcp-filesystem-server
 eterna % /Users/XX/go/bin/mcp-filesystem-server /Users/XX/eterna /Users/XX/eterna/pdfs
